docs/home/howto/kubernetes/PERSISTENT_STORAGE.md

```diff
@@ -90,6 +90,24 @@ specification:
 
 #### Create disks for Rook/Ceph Cluster Storage - AWS
 
+To define additional empty disk resources for Rook/Ceph Cluster Storage on AWS, use `specification.disks.additional_disks` under `kind: infrastructure/virtual-machine` for configuration of kubernetes node machine:
+```yaml
+---
+kind: infrastructure/virtual-machine
+title: Virtual Machine Infra
+provider: aws
+name: kubernetes-node-machine
+specification:
+  disks:
+    additional_disks:
+      - volume_type: gp2
+        volume_size: 64
+        delete_on_termination: false
+        encrypted: true
+```
+More information about AWS block devices: [ebs_block_device](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#ebs_block_device)
```

Comment:
Maybe it will be good to write that parameters for additional_disk are taken from AWS block devices and list supported parameters?